src/Components/ArticleCard.jsx

```diff
@@ -25,10 +38,31 @@ export default function ArticleCard({ article }) {
   }
 
   return (
-    <section>
+    <><section>
       <div className="articleName">
-        
+        <p>{article.title}</p>
```

Comment:
if you have a title, maybe it's good to use a <h2> tag or something similar to make this accessible